src/App/Utilities/PasswordFormatter.cs

```diff
@@ -56,24 +54,39 @@ public static FormattedString FormatPassword(string password)
                 // If the char type changed, build a new span to append the text to.
                 if(charType != currentType)
                 {
-                    currentSpan = new Span();
-                    result.Spans.Add(currentSpan);
+                    // Close off previous span.
+                    if (currentType != CharType.None)
```

Comment:
Shouldn't this be `currentType != CharType.None || currentType != CharType.Normal`?

Reply:
Indeed. Except && instead of ||